api/dp-openapi.yaml

```diff
@@ -1020,6 +1189,69 @@ components:
       items:
         type: string
 
+    proof_page_text:
+      type: object
+      properties:
+        text:
+          description: the page text from current round.
+          type: string
+
+    proof_page_data:
+      type: object
+      properties:
+        text:
+          description: the page text. If page state is 'out' from OCR or previous round, else from current round
+          type: string
+        pagestate:
+          description: the page state, for checking continuity. Either out or temp
+          type: string
+        saved:
+          description: true if the page state is 'temp'
+          type: boolean
+        pagename:
+          description: the filename of the page with extension
+          type: string
+        image_url:
+          description: url of the image of the page
+          type: string
+        pageNum:
```

Comment:
I was, unfortunately, inconsistent in how I named properties -- sometime I used an `_` and sometime I didn't. That's on me. But let's not introduce snakeCase as a third. Suggest this be `pagenum` to match `pagename`.

Reply:
Why do we want this separate from pagename?

Reply:
changed to pagenum. This is how the existing PI shows the page number (in the text below the buttons in standard interface) Should we keep it like this?